src/index-node.ts

```diff
@@ -28,9 +28,21 @@ export class SiweClient extends SiweImpl {
 
   async _extractCookies(headers?: Headers): Promise<void> {
     const cookieRecord: Record<string, string> = {}
-    const headerSetCookies = (headers as any).raw()[
-      'set-cookie'
-    ] as Array<string>
+    // previously
+
+    let headerSetCookies: Array<string> = []
+
+    try {
+      // returns headers.raw is not a function while running the sdk on a repl
```

Comment:
is that because headers is undefined? if that's the case, it doesn't seem like the logic in the `catch` block would ever get used